Kickstarter-iOS/Views/Cells/ProjectPamphletCreatorHeaderCell.swift

```diff
@@ -0,0 +1,134 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+private enum Layout {
+  enum Button {
+    static let height: CGFloat = 48
+    static let width: CGFloat = 152
+  }
+}
+
+protocol ProjectPamphletCreatorHeaderCellDelegate: class {
+  func projectPamphletCreatorHeaderCellDidTapButton(
+    _ cell: ProjectPamphletCreatorHeaderCell,
+    project: Project
+  )
+}
+
+final class ProjectPamphletCreatorHeaderCell: UITableViewCell, ValueCell {
```

Comment:
Still not sure about this name 🤔  Would `ProjectPamphletViewProgressCell` be better?